examples/cpp/helloworld/greeter_server.cc

```diff
@@ -47,19 +59,55 @@ class GreeterServiceImpl final : public Greeter::Service {
     return Status::OK;
   }
 };
+*/
 
 void RunServer() {
   std::string server_address("0.0.0.0:50051");
-  GreeterServiceImpl service;
 
   grpc::EnableDefaultHealthCheckService(true);
   grpc::reflection::InitProtoReflectionServerBuilderPlugin();
+
   ServerBuilder builder;
+
   // Listen on the given address without any authentication mechanism.
   builder.AddListeningPort(server_address, grpc::InsecureServerCredentials());
+
   // Register "service" as the instance through which we'll communicate with
   // clients. In this case it corresponds to an *synchronous* service.
-  builder.RegisterService(&service);
+
+  // GreeterServiceImpl service;
+  // builder.RegisterService(&service);
+
+  const std::string& path = "other_service_shared.dll"; 
+  // const std::string& path = "C:\\Users\\keithl\\repos\\grpc\\examples\\cpp\\helloworld\\cmake\\build\\Debug\\other_service_shared.dll"; 
```

Comment:
Note that this should either be a relative path or rename to wherever your DLL lives